test/test_functionalization.py

```diff
@@ -1639,6 +1639,14 @@ def forward(self, arg0_1):
     return add""")
         self.assertEqual(fx_g_cpp.code.strip(), fx_g.code.strip())
 
+    def test_python_functionalization_lift_fresh_storage(self):
+        unlifted = torch.tensor([0.0])
```

Comment:
Hmm, can you make this test a bit more complicated? e.g.:
```
def f(x):
    tmp = torch.tensor([0.0])
    return x + tmp
```

That will at least convince me of whether the change for `lift_fresh` below works properly :). There's an invariant that if an output to an op is an `at::functionalization::FunctionalTensorWrapper`, then python functionalization needs to wrap the output in a python `FunctionalTensor` subclass.